ChronoKeeper/StoryChunkExtractor.cpp

```diff
@@ -96,11 +96,13 @@ void chronolog::StoryChunkExtractorBase::drainExtractionQueue()
                 LOG_WARNING("[StoryChunkExtractionBase] Failed to acquire a story chunk from the queue.");
                 break;
             }
-            processStoryChunk(storyChunk);  // INNA: should add return type and handle the failure properly
+            int ret = processStoryChunk(storyChunk);  // INNA: should add return type and handle the failure properly
+            LOG_DEBUG("[StoryChunkExtractionBase] Story chunk processing returns {}, ES Rank: {}, ULT "
```

Comment:
in case processStoryChunk fails the StoryChunk should be pushed back on ExtractionQueue so that it's not lost..